src/core.js

```diff
@@ -212,7 +212,10 @@ jQuery.extend( {
 	noop: function() {},
 
 	isFunction: function( obj ) {
-		return jQuery.type( obj ) === "function";
+
+		// Support: Chrome <=57, Firefox <=52
```

Comment:
Should we add `only` to both or not yet because Chrome 58 & Firefox 53 are not stable yet?